samples/nrf_desktop/src/events/led_event.c

```diff
@@ -14,16 +14,26 @@ static const char * const mode_name[] = {
 #undef X
 };
 
-static void print_event(const struct event_header *eh)
+static size_t log_led_event(const struct event_header *eh, char *buf,
+			  size_t buf_len)
 {
 	struct led_event *event = cast_led_event(eh);
+	size_t temp;
+	size_t pos = 0;
+
+	temp = snprintf(buf, buf_len, "led_id:%u mode:%s period:%u color: <",
```

Comment:
you must make sure there is no buffer overflow - check `snprintf` documentation

Reply:
done